tests/storages_tests/journal_tests/assets/4.0.0.dev.log

```diff
@@ -0,0 +1,92 @@
+{"op_code": 0, "worker_id": "a4572293-4260-45be-9d2a-489e532dc113-140704507377600", "study_name": "single_empty", "directions": [1]}
```

Comment:
Just for the record, I confirmed that the asset log file contains [all kinds of `op_type`](https://github.com/optuna/optuna/blob/6aa86fc37811a421d444b6426eae1fc7c544ea96/optuna/storages/_journal/storage.py#L39-L49) 👍 

```python
>>> import json
>>> with open("tests/storages_tests/journal_tests/assets/4.0.0.dev.log") as f:
...     lines = f.readlines()
...
>>> d = [json.loads(l) for l in lines]
>>> set([a["op_code"] for a in d])
{0, 1, 2, 3, 4, 5, 6, 7, 8, 9}
```